redbot/cogs/downloader/repo_manager.py

```diff
@@ -1119,21 +1122,22 @@ def get_all_cogs(self) -> Tuple[Installable, ...]:
         Tuple[Repo, Tuple[str, str]]
             A 2-`tuple` with Repo object and a 2-`tuple` of `str`
             containing old and new commit hashes.
-
         """
         repo = self._repos[repo_name]
         old, new = await repo.update()
         return (repo, (old, new))
 
     async def update_repos(
-        self, upd_repo_names: List[Repo] = []
+        self, upd_repos: Optional[List[Repo]] = None
```

Comment:
That was there because there were already lots of repo, repos, Repo and alike used inside.
Tried to say "repos_to_update". Looking around, I should indeed turn it around and change the param to `repos` and if needed change the naming inside the func.